Makefile

```diff
@@ -400,10 +405,19 @@ $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TAR
 	| $(BUILD_OUTPUT_DIRECTORY)
 	./scripts/build/zip-file.sh -f $< -s $(TARGET_PLATFORM) -o $@
 
-$(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH).exe: \
-	$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH) \
-	| $(BUILD_OUTPUT_DIRECTORY) $(BUILD_TEMPORARY_DIRECTORY)
-	./scripts/build/electron-installer-nsis-win32.sh -n $(APPLICATION_NAME) -a $< -t $(BUILD_TEMPORARY_DIRECTORY) -o $@
+ELECTRON_BUILDER_OPTIONS=--$(ELECTRON_BUILDER_TARGET_ARCH) --extraMetadata.version=$(APPLICATION_VERSION)
+
+ifdef ANALYTICS_SENTRY_TOKEN
```

Comment:
I wonder if it might be worth removing the blank lines at https://github.com/resin-io/etcher/blob/0b067495c636c19bf934ddda28b07683e9c367db/Makefile#L228 and https://github.com/resin-io/etcher/blob/0b067495c636c19bf934ddda28b07683e9c367db/Makefile#L236 to make it clearer that they're all part of the same make-rule, and not actually separate rules? I had to read it a few times before I realised what was going on...